xsd/netex_framework/netex_reusableComponents/netex_serviceCalendar_version.xsd

```diff
@@ -499,12 +499,7 @@ Rail transport, Roads and Road transport
 		<xsd:sequence>
 			<xsd:element name="ValidDayBits" type="xsd:normalizedString">
 				<xsd:annotation>
-					<xsd:documentation>String of bits, one for each day in period: whether valid or not valid on the day.  Normally there will be a bit for every day between start and end date.  If bit is missing, assume available.</xsd:documentation>
-				</xsd:annotation>
-			</xsd:element>
-			<xsd:element name="DaysOfWeek" type="DaysOfWeekListOfEnumerations" minOccurs="0">
-				<xsd:annotation>
-					<xsd:documentation>Days of week to which bits correspond.</xsd:documentation>
+					<xsd:documentation>String of bits, one for each day in period: whether valid or not valid on the day. Normally there will be a bit for every day between start and end date.  If bit is missing, assume available.</xsd:documentation>
```

Comment:
```suggestion
					<xsd:documentation>String of bits, one for each day in period: whether valid or not valid on the day. Normally there will be a bit for every day between start and end date. If bit is missing, assume available.</xsd:documentation>
```
I think I also want to remove the "If bit is missing" part. Because why allow organisations to actively provide bad information?

Reply:
I made a bet with me that you would correct it :-)

Reply:
You should have placed a bigger bet ;-)